site/components/form/Select.tsx

```diff
@@ -49,11 +53,20 @@ const Select = <T extends object>({
                 </label>
                 <FormElementWrapper errors={errors} errorId={inputName} errorClass="govuk-select--error">
                     <select
-                        className="govuk-select w-3/4"
+                        className={`govuk-select w-${width}`}
                         name={inputName}
                         id={`${inputId}-input`}
-                        defaultValue={value ?? ""}
```

Comment:
This should stay as defaultValue for the form pages that are using this component so we may need to toggle based on the context

Reply:
![Screenshot 2023-05-05 at 16 11 36](https://user-images.githubusercontent.com/111580926/236497554-bfdc6622-9364-43d8-b786-c4de2e0100cf.png)

Is this what you mean? Feels like its almost correct but something is off.

Reply:
This should work but it'll need defaulting to true rather than false and overriding on the view all disruptions page

Reply:
This should work but it'll need to be defaulted to true and then overridden to false on the view all disruptions page